sources/utils/Utils.swift

```diff
@@ -73,3 +73,5 @@ func ==(lhs: [[JSON]], rhs: [[JSON]]) -> Bool {
     }
     fatalError("Incorrect equality operator implementation!")
 }
+
+let testUrl = URL(string: "http://test.com")!
```

Comment:
bad idea to add a global variable even for test. You can override already existing data

Reply:
Check player core.